roles/openshift_logging_curator/defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 ### General logging settings
 openshift_logging_curator_image_prefix: "{{ openshift_hosted_logging_deployer_prefix | default('docker.io/openshift/origin-') }}"
-openshift_logging_curator_image_version: "{{ openshift_hosted_logging_deployer_version | default('latest') }}"
+openshift_logging_curator_image_version: "{{ openshift_logging_image_version }}"
```

Comment:
Not yet, we need to mark them deprecated in 3.7 and remove them in 3.8.
https://trello.com/c/XKOGHSkP/460-3-deprecate-openshifthostedloggingmetrics-installer-variables